src/main/java/li/cil/tis3d/client/gui/ReadOnlyMemoryModuleGui.java

```diff
@@ -241,7 +241,7 @@ private void drawHeaders() {
         GlStateManager.pushMatrix();
         GlStateManager.translatef(guiX + GRID_LEFT + 3, guiY + 6, 0);
         for (int col = 0; col < 16; col++) {
-            FontRendererAPI.drawString(String.format("%X", col));
+            SmallFontRenderer.INSTANCE.drawString(String.format("%X", col));
```

Comment:
Keep dogfooding here (and below), i.e. `LocalAPI.client.fontRendererAPI.drawString(ClientAPI.Font.SmallFont, ...)`?

Reply:
Will do! I think at the time I changed this, the font renderer API was being changed around a lot / very unstable, so I just wanted this to work for the time being.

Reply:
Right, so in reference to the comment further above, I think the intention here was to temporarily couple all of the font-using UI code to the internal classes for now, and when they're rewritten to use the VertexConsumer-based classes, the dogfooding can be reinstated. Thoughts?

Reply:
Oh, sure, we can hold off on this until that, then.